libs/sdk-ui-kit/src/AppearancePage/StylingHeader.tsx

```diff
@@ -0,0 +1,9 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+
+/**
+ * @internal
+ */
+export const StylingHeader: React.FC = ({ children }) => {
```

Comment:
I would rename this component just to `Header`. `DocumentationLink` and `TooltipIcon` does not have any prefix either.